README.md

```diff
@@ -19,9 +19,17 @@ docker build -t text-classifier:latest .
 --port (-p)              port to use (default 9009)
 --threads (-t)           number of threads (default 1)
 --debug (-d)             debug mode (default false)
+--grpc (-g)              use grpc service instead of rest
```

Comment:
Also add [--grpc] in .usage() in text-classifier.cpp

Reply:
Corrections asked done